9.5-2.3/README.md

```diff
@@ -27,3 +27,20 @@ Once you have started a database container, you can then connect to the database
         sh -c 'exec psql -h "$POSTGRES_PORT_5432_TCP_ADDR" -p "$POSTGRES_PORT_5432_TCP_PORT" -U postgres'
 
 See [the PostGIS documentation](http://postgis.net/docs/postgis_installation.html#create_new_db_extensions) for more details on your options for creating and using a spatially-enabled database.
+
+When You encouter errors due to PostGIS update `OperationalError: could not access file "$libdir/postgis-$X`, run:
+
+`docker exec some-postgis /update-postgis.sh`
```

Comment:
Assuming the script is moved to `/usr/local/bin`, this can be `update-postgis.sh` instead of `/update-postgis.sh`.